src/lib.rs

```diff
@@ -55,7 +55,7 @@ mod ffi_utils;
 #[cfg_attr(target_os = "aix", path = "tz_aix.rs")]
 #[cfg_attr(target_os = "android", path = "tz_android.rs")]
 #[cfg_attr(target_os = "haiku", path = "tz_haiku.rs")]
-mod platform;
+mod tz_darwin;
```

Comment:
This looks like a bad editor refactor. This module must always be named platform. Its contents are determined by the `#[path]` attribute in the chain of cfg above.

Reply:
Oops, indeed, `rust-analyzer` must've done it behind my back